### ISSUE TITLE
Reporter Not Found since this morning

### ISSUE THREAD

**Issue by chrisvidal** [adamgruber/mochawesome#71]
Since this morning, i got this error when running
with mochawesome 1.4.0

````bash
mocha --timeout 5000 tests/stats --reporter mochawesome
"mochawesome" reporter not found
/Users/qwerty/npm/lib/node_modules/mocha/lib/mocha.js:164
      throw new Error('invalid reporter "' + reporter + '"');
      ^

Error: invalid reporter "mochawesome"
    at Mocha.reporter (/Users/qwerty/npm/lib/node_modules/mocha/lib/mocha.js:164:13)
    at Object.<anonymous> (/Users/qwerty/npm/lib/node_modules/mocha/bin/_mocha:207:7)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
````

**Comment by adamgruber** [comment]
Can you double-check the version? I released 1.5.0 yesterday so maybe there's an issue with that. What version of mocha are you running? Also, some test code would help. I just put together a small test and it was working ok for me.

**Comment by chrisvidal** [comment]
the mochawesome version installed locally is the 1.4.0
i solved the problem with installing mocha locally.
the previous error happen with a mocha installed globally.
not sure why, but it looks like it comes from my global mocha installation
